README.md

```diff
@@ -432,6 +433,79 @@ Example 3:
 
 _________________________
 
+
+### qLoRA Tuning Example
+
+This method is similar to LoRA Tuning, but the base model is a quantized model.
```

Comment:
Would be useful to add a note pointing to the fms-acceleration section and note that is how qlora is enabled - https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/README.md#fms-acceleration